cal/cal/models.py

```diff
@@ -620,7 +620,7 @@ def save(self, *args, **kwargs):
     def hours(self, calendar_ids=None, start=None, end=None):
         events = self.get_events(calendar_ids=calendar_ids, start=start, end=end)
 
-        return EventCollection(lambda: events).total_time() / 3600
+        return round(float(EventCollection(lambda: events).total_time()) / 3600, 2)
```

Comment:
Could you perhaps add a comment here explaining why this is needed? For me at least it was not readily apparent